src/man/sssd.conf.5.xml

```diff
@@ -2311,6 +2311,19 @@ p11_uri = library-description=OpenSC%20smartcard%20framework;slot-id=2
                             You can consider setting this value to
                             3/4 * entry_cache_timeout.
                         </para>
+                        <para>
+                            Cache entry will be refreshed by background task
+                            when 2/3 of cache timeout has already passed.
+                            If there are existing cached entries, the background
+                            task will refer to their original cache timeout
+                            values instead of current configuration value.
+                            This may lead to situation when background refresh
+                            task looks to be not working. This is done
```

Comment:
I'd suggest changing "looks" to "appears" or "seems"   and change order to "to not be".

Something like this:

"task appears to not be working"

Reply:
Fixed